wire-ios-data-model/Source/Model/Conversation/ZMConversation+MLS.swift

```diff
@@ -197,7 +197,7 @@ public extension ZMConversation {
             argumentArray: [Self.mlsGroupIdKey, groupID.data]
         )
 
-        let result = context.executeFetchRequestOrAssert(request)
+        let result = try! context.fetch(request)
```

Comment:
Do I understand correctly the objc method also crashes on any error? So the behavior hasn't changed?

Reply:
So it looks like it the previous method also crashes upon any error. I'm not sure how I feel though about crashing, but since it's not a change in behavior I'm fine to discuss it outside of the PR, and at least now it's more obvious that there is a crash.

Reply:
> at least now it's more obvious that there is a crash.

I totally agree, this now catches everybody's eye.